docs/layers.md

```diff
@@ -2288,6 +2288,7 @@ However, if the roads layer does not include `linear_ref_id`, then it should be
 * `is_bridge`: set to `true` when the linear is a bridge
 * `is_tunnel`: set to `true` when the linear is a tunnel
 * `is_link`: set to true when the linear is a slip-road
+* `is_express_lane`: set to `true` when the linear is an HOV (High-Occupancy Vechicle) lane.
```

Comment:
Why is this necessary, can you explain your rational with an example, please? 

We currently don't include `hov` tags on the **roads** layer features, so seems slightly odd to include it here in the **traffic_flow** and **traffic_incidents** layers. While there has been discussion to add lane tags into the tiles, and then hov could be added as part of that, I'm still not sure why it's necessary – except for HD mapping, which Tilezen currently isn't really geared for.

If we were to do this...

The OSM-ism we'd follow is `hov` instead of `express`, please say `is_hov` instead.

See also:

- https://wiki.openstreetmap.org/wiki/Key:hov

Visually would you sort the HOV traffic over the non-HOV traffic flow lines? Would you want a way to turn off the non-HOV traffic lines? This starts to get tricky fast.

Reply:
Hi Nathaniel, 
I got confused (I'm British/Irish) and misunderstood what an Express Lane is. So updating the patch with a new commit for the explanations.

However this still won't satisfy this comment:
> While there has been discussion to add lane tags into the tiles, and then hov could be added as part of that, I'm still not sure why it's necessary – except for HD mapping, which Tilezen currently isn't really geared for.

Indeed this is lane-specific information, but basically we'd like to be able to have this tag so that the client is aware that the Traffic Flow/Incident information is affecting the Express Lane on a particular road.